library/src/test/java/com/datatorrent/lib/io/fs/AbstractFileOutputOperatorTest.java

```diff
@@ -1756,11 +1766,14 @@ public void testChainFilters() throws NoSuchAlgorithmException, IOException
       for (int j = 0; j < 1000; ++j) {
         writer.input.put(i);
       }
-      writer.endWindow();
-      evenOffsets.add(evenCounterContext.getCounter());
-      oddOffsets.add(oddCounterContext.getCounter());
-      //evenOffsets.add(evenFile.length());
-      //oddOffsets.add(oddFile.length());
+      //writer.endWindow();
```

Comment:
why commenting endWindow()? engine calls endWindow() before `beforeCheckpoint()` callback so there isn't any need to comment this out.